api/openapi-spec/swagger.json

```diff
@@ -8106,8 +8106,21 @@
     }
    },
    "v1.Patch": {
-    "description": "Patch is provided to give a concrete name and type to the Kubernetes PATCH request body.",
-    "type": "object"
+    "type": "object",
+    "properties": {
+     "patch": {
+      "type": "string"
+     },
+     "resourceName": {
+      "type": "string"
+     },
+     "resourceType": {
+      "type": "string"
+     },
+     "type": {
+      "type": "string"
+     }
+    }
```

Comment:
Is this related ?

Reply:
Nope, dropping

Reply:
this is the flaky generate causing it, dont try to fight it, its WIP